static/locales/en-US.yaml

```diff
@@ -145,7 +146,16 @@ Settings:
     Clear Default Instance: Clear Default Instance
     View all Invidious instance information: View all Invidious instance information
     Region for Trending: Region for Trending
-        #! List countries
+    #! List countries
+    External Link Handling:
+      External Link Handling: External Link Handling
+      External Link Handling Tooltip: |
+        External Link Handling: Choose the behaviour when an external link (a link with non Youtube URL) is clicked.
```

Comment:
behavior* for American English. Sorry, another final nitpick.

Reply:
I'm also not sure if "non YouTube url" would translate well. I think something like "a link that opens in your default browser" would be more translatable.

Reply:
> `behavior`

I can never remember which one is US and which one is UK variant @@

Updated text to 
>
        External Link Handling: Choose the behavior when a link is clicked.
        By default FreeTube will open the link in your default browser.
        (Unless it points to a Youtube Video/Playlist/Channel etc.)
        You can choose to disable it completely for safety.

I think this setting is controlling **any** link except those pointing to Youtube "resources".
Though I am not sure if this updated text can be easily comprehended and translated.